states/IN/governor.yaml

```diff
@@ -0,0 +1,212 @@
+            "Rev.": "Rev."
+            "Reverend": "Reverend"
+            "Sir": "Sir"
+            "Sister": "Sister"
+            "The Honorable": "The Honorable"
+    - fill_in:
+        - name: "first"
+          selector: "#first"
+          value: $NAME_FIRST
+          required: true
+          # max length: 25
+        - name: "last"
+          selector: "#last"
+          value: $NAME_LAST
+          required: true
+          # max length: 25
+        - name: "street"
+          selector: "#street"
+          value: $ADDRESS_STREET
+          required: true
+          # max length: 50
+        - name: "city"
+          selector: "#city"
+          value: $ADDRESS_CITY
+          required: true
+    - select:
+        - name: "state"
+          selector: "select[name='state']"
+          value: $ADDRESS_STATE_POSTAL_ABBREV
+          required: true
+          options:
+            "AL": "Alabama"
+            "AK": "Alaska"
+            "AZ": "Arizona"
+            "AR": "Arkansas"
+            "CA": "California"
+            "CO": "Colorado"
+            "CT": "Connecticut"
+            "DC": "Dist. of Col."
+            "DE": "Delaware"
+            "FL": "Florida"
+            "GA": "Georgia"
+            "HI": "Hawaii"
+            "ID": "Idaho"
+            "IL": "Illinois"
+            "IN": "Indiana"
+            "IA": "Iowa"
+            "KS": "Kansas"
+            "KY": "Kentucky"
+            "LA": "Louisiana"
+            "ME": "Maine"
+            "MD": "Maryland"
+            "MA": "Massachusetts"
+            "MI": "Michigan"
+            "MN": "Minnesota"
+            "MO": "Missouri"
+            "MS": "Mississippi"
+            "MT": "Montana"
+            "NE": "Nebraska"
+            "NV": "Nevada"
+            "NH": "New Hampshire"
+            "NJ": "New Jersey"
+            "NM": "New Mexico"
+            "NY": "New York"
+            "NC": "North Carolina"
+            "ND": "North Dakota"
+            "OH": "Ohio"
+            "OK": "Oklahoma"
+            "OR": "Oregon"
+            "PA": "Pennsylvania"
+            "RI": "Rhode Island"
+            "SC": "South Carolina"
+            "SD": "South Dakota"
+            "TN": "Tennessee"
+            "TX": "Texas"
+            "UT": "Utah"
+            "VT": "Vermont"
+            "VA": "Virginia"
+            "WA": "Washington"
+            "WV": "West Virginia"
+            "WI": "Wisconsin"
+            "WY": "Wyoming"
+            "AS": "American Samoa"
+            "FM": "Fed. States Micronesia"
+            "GU": "Guam"
+            "MH": "Marshall Islands"
+            "MP": "North Mariana Is."
+            "PR": "Puerto Rico"
+            "VI": "Virgin Islands"
+            "Other": "Other (Use Address Lines)"
+    - fill_in:
+        - name: "zip"
+          selector: "#zip"
+          value: $ADDRESS_ZIP5
+          required: true
+          # max length: 5
+        - name: "phone"
+          selector: "#phone"
+          value: ""
```

Comment:
Is this just in case we want to include phone numbers in the future, or was this a mistake?

Reply:
Ah that sounds like a good approach! Maybe add a comment memorializing that.

Reply:
I made a comment in the code as such.